flink-python/pyflink/table/descriptors.py

```diff
@@ -33,7 +33,6 @@
     'FileSystem',
     'Kafka',
     'Elasticsearch',
-    'HBase',
```

Comment:
Does this mean, you cannot use HBase from Python anymore? Why is it different than the other connectors?

Reply:
You can still use the SQL DDL for using HBase and this is also the officially documented way. The `connect()` API is outdated and requires an update (FLIP-129). I guess, initially, the Python team just adopted all "descriptors" that were present in 2019.